news/2024/2024-06-24-4-digit-world-cup-2024-concludes.md

```diff
@@ -0,0 +1,167 @@
+> 
+> Nonetheless, teams do get high seedings not without a merit. I don't think upsets are something that will happen more often but only as often as or less than it is now. If upsets do spike up in quantity, it is only because of an erroneous early round pooling that deviates completely from the qualifier (Ex. speed qualifier into full gimmick RO32).
+> 
+> Of course, its frequency is not what makes an upset exciting. We all love an underdog story, the David against the Goliath, and this is no exception especially when it comes to a tiebreaker. At that point, the audience and the casters have won regardless of the result.
+
+::{#shurelia}::
+
+[![](/wiki/shared/news/2024-06-24-4-digit-world-cup-2024-concludes/shurelia-banner.png)](https://osu.ppy.sh/users/3807986)
+
+> Before I start my introduction, I would love to share my gratitude towards the 4WC 2024 staff for the opportunity to let me be involved with their event, and also for the participants and their positive feedback towards my maps which boosts my confidence on this. I thank you, really.
+> 
+> I'm Shurelia, a catch player (despite being involved in mapping for standard tournaments) since 2012 and also a GMT since 2014. I've been actively mapping for standard since 2014 and finally got into tournament mapping in 2023.
+
+**Shurelia** has started, despite being an osu!catch player, making tournament maps for osu! recently, including a total of three for this year's 4WC.
+
+---
+
+*What brought you into mapping? Why have you decided to create custom maps for tournaments such as 4WC 2024?*
+
+> The reason why I started mapping for osu! standard specifically was due to being a heavy convert player in catch and wanting to gain #1 on a song that I really like. But the reason why I WANT to be better in mapping is because I got captivated by [Lan Wing](https://osu.ppy.sh/users/467860)'s beatmap of [t=NODE - Four Seasons](https://osu.ppy.sh/beatmapsets/285718#osu/645242). This map moves me so much that I randomly shed tears despite having no clue about osu! standard at all due to how beautiful the map and the song is.
+> 
+> I decided to create custom maps because I just love watching people playing my maps, especially when it's in a big tournament such as 4WC 2024. It also lets me learn many things to improve my mapping knowledge gameplay wise by learning what the players think about my map when they try it, because as I mentioned, I do not play osu! standard at all and thus it's obvious that I don't testplay my own maps at all. Custom mapping is such a good way to improve myself, at least that's what I'm thinking.
```

Comment:
i think its fair to say that testing a map *is* playing it so not sure if the original wording is valid, this is kinda subjective though i dont have a strong opinion on this
```suggestion
> I decided to create custom maps because I just love watching people playing my maps, especially when it's in a big tournament such as 4WC 2024. It also lets me learn many things to improve my mapping knowledge gameplay wise by learning what the players think about my map when they try it, because as I mentioned, I do not play osu! standard at all and thus it's obvious that I don't test my own maps at all. Custom mapping is such a good way to improve myself, at least that's what I'm thinking.
```

Reply:
mapper friend said testplay makes it a lot more clear that its specifically *playing* and not something else so keeping as is